tests/test-daemon-integration.py

```diff
@@ -31,9 +31,8 @@
 
 class TestDaemonIntegration(dbusmock.DBusTestCase):
     _MOCK_EVENT_NOTHING_HAPPENED = '5071dd96-bdad-4ee5-9c26-3dfef34a9963'
-    _MOCK_EVENT_NOTHING_HAPPENED_BYTES = \
-        uuid.UUID(_MOCK_EVENT_NOTHING_HAPPENED).bytes
-    _NANOSECONDS_PER_SECOND = 1000000000L
+    _MOCK_EVENT_NOTHING_HAPPENED_UUID = \
+        uuid.UUID(_MOCK_EVENT_NOTHING_HAPPENED)
```

Comment:
Can you just put the UUID string constant in here, since you shouldn't need to reuse it?